pattern-2/bosh-release/jobs/health_check/templates/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+health_check() {
+    # Check if health check endpoint is alive
+    if curl --output /dev/null --silent --fail -k "$1"
+    then
+        status_code=$(curl --write-out %{http_code} --silent --output /dev/null -k ${1})
+
+        # Check if requests to the health check endpoint produces  200 response
+        if [[ "$status_code" -ne 200 ]] ; then
+            echo "WSO2 APIM $2 produces an invalid response: $status_code" >>/dev/stderr
+            exit 1
+        else
+            echo "WSO2 APIM $2 is Running!"
+        fi
+    else
+        echo "WSO2 APIM $2 is not alive" >>/dev/stderr
+        exit 1
+    fi
+}
+
+carbonHealthCheckEP="https://localhost:9443/carbon/admin/login.jsp"
+
+health_check ${carbonHealthCheckEP} Carbon
+exit 0
```

Comment:
Missing empty line at the end of the file.

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c